src/BaseballGame.js

```diff
@@ -0,0 +1,74 @@
+import { MissionUtils } from "@woowacourse/mission-utils";
+import { generateRandomNum } from "./utils/generateRandomNum";
+import { isInputValid, isEndInputValid } from "./utils/validation";
+import { getResult, printResult } from "./utils/getResult";
+
+class BaseballGame {
+  #COMPUTER_NUM_LEN = 3;
+  #ComputerNums;
+  #UserNums;
+
+  setComputerNums(nums) {
+    this.#ComputerNums = nums;
+  }
+
+  setUserNums(nums) {
+    isInputValid(nums, this.#COMPUTER_NUM_LEN);
+    this.#UserNums = nums.split("").map((str) => Number(str));
+  }
+
+  /**
+   * 게임의 전체 진행을 위한 메서드입니다.
+   */
+  async runGame() {
+    await this.startGame();
+    await this.playGame();
+    await this.endGame();
+  }
+
+  /**
+   * 게임 시작을 위한 메서드입니다.
+   * 게임 시작 멘트를 출력하며, 컴퓨터의 숫자 3개를 정합니다.
+   */
+  async startGame() {
+    await MissionUtils.Console.print("숫자 야구 게임을 시작합니다.");
```

Comment:
```suggestion
    await MissionUtils.Console.print("숫자 야구 게임을 시작합니다.");
```
`MissionUtils.Console.print` 메서드는 Promise를 반환하지 않는 동기 함수여서 `await`에 영향을 받지 않아서 제거해줘도 좋을 것 같아요!

Reply:
헐랭... `MissionUtils.Console.readLineAsync`땜에 한번에 다 `await`를 넣어준 것 같네요ㅠㅠ 좀 더 꼼꼼하게 확인해볼걸 그랬습니다. 감사드립니다!
혹시 `MissionUtils.Console.print`가 동기함수인건 어떻게 아셨는지 여쭤봐도 될까요?

Reply:
`node_modules/@woowacourse/mission-utils/src/console.js` 라이브러리 패키지 내부 코드에 `print` 함수는 Promise를 반환하지 않고 단순한 `console.log` 메서드인걸로 확인했습니다!

```js
class Console {
  constructor() {}

  static print(message) {
    console.log(message);
  }
}
```

Reply:
:open_mouth: 와우 감사합니다!